lib/logstash/filters/elasticsearch.rb

```diff
@@ -84,7 +84,7 @@ def filter(event)
     begin
       query_str = event.sprintf(@query)
 
-      params = { :q => query_str, :size => result_size }
+      params = { :q => query_str, :size => result_size, :index => @index }
```

Comment:
1.- It looks like is not being applied as expected as I initially thought, I will have to look closer to understand more. *I agree if does not work as expected should go*, removing this.

2.- yes, https://github.com/elastic/elasticsearch-ruby/blob/master/elasticsearch-api/lib/elasticsearch/api/actions/search.rb#L65 they are the same.